xbmc/interfaces/legacy/ListItem.h

```diff
@@ -654,6 +656,8 @@ namespace XBMCAddon
       /// Added labels **setid**, **set**, **imdbnumber**, **code**, **dbid**, **path** and **userrating**.
       /// Expanded the possible infoLabels for the option **mediatype**.
       /// @python_v18 Added new **game** type and associated infolabels.
+      /// Added labels **setoverview**, **tag**.
```

Comment:
@AlwinEsch does every line here need a separate python_v18 tag so that they get indexed correctly?